File: Version0.3/Matlab/Preprocessing/meshpartition2.m
function dmd = meshpartition2(t,f,t2t,bcm,dim,elemtype,porder,nproc,metis)

disp('run elementpartition...');  
dmd = elementpartition2(t,t2t,nproc,metis);

disp('run facepartition...');  
dmd = facepartition2(dmd,t,f,bcm,dim,elemtype,porder,nproc);

File: Version0.1/Julia/Preprocessing/meshpartition2.jl
function meshpartition2(dmd,t,f,t2t,bcm,dim,elemtype,porder,nproc,metis)

print("run elementpartition...\n");
dmd = elementpartition2(dmd,t,t2t,nproc,metis);

print("run facepartition...\n");
dmd = facepartition2(dmd,t,f,bcm,dim,elemtype,porder,nproc);

return dmd;

end
